app/views/job_templates/index.html.erb

```diff
@@ -1,4 +1,4 @@
-<%= include_javascript %>
+<%= include_javascript if SETTINGS[:version].short == '1.9' %>
```

Comment:
I agree with you in general, although I don't think cleaning the checks after we say that new version will be not 1.9 compatible is hard thing to do. We agreed on keeping the compatibility with the last stable release until there is something we need in core to support our feature, or there is some big change so that supporting both old and new versions would be inefficient. I don't think any of this changes fit into this category: this are really minor things. Being compatible with latest stable has also it's advantages: you can notice that we don't hit CI errors in the development process, that are caused by temporary breakages in nightlies: so we have time to work on compatibility with new version, while the development can continue without issues on 1.9 stable (I've seen many times this causing issues in Katello when running against nightlies).

We can talk about switching compatible versions at the end of iteration 2, so far, I would keep compatibility with 1.9.